source/topic6.rst

```diff
@@ -202,13 +202,48 @@ Push
 Pop and Peek
 ^^^^^^^^^^^^
 
+* ``pop`` and ``peek`` will be similar, except peek leaves the top alone
 
 
+.. code-block:: Java
+    :linenos:
+    :emphasize-lines: 1, 2, 3, 4, 11, 12, 13, 14
 
+        public T pop() throws EmptyStackException {
```

Comment:
Had them because text uses them, but if anything I think it could be more confusing here since they are not that used to exceptions at this stage. 

In other words, good call.